scripts/core/ui/components/utils.ts

```diff
@@ -19,15 +19,31 @@ export const onEventCapture = (event) => {
  * @name gettext
  * @param {String} text
  * @param {Object} params
+ * @param {String} plural
+ * @param {number} count required if plural is present
  * @description Used angular gettext service for displaying localised text on Browser
+ *
+ * Examples:
+ *
+ * gettext('This item was locked by {{username}}.', {username: 'John'});
+ * result -> 'This item was locked by John'
+ *
+ * gettext('One item.', {}, '{{$count}} items', 1);
+ * result -> 'One item'
+ *
+ * gettext('One item.', {}, '{{$count}} items', 2);
+ * result -> '2 items'
  */
-export const gettext = (text, params = null) => {
+export const gettext = (text, params = null, plural = null, count = null) => {
```

Comment:
Looks like the extraction of gettext with plural is not supported.
I think we have to use gettextCatalog.getPlural for this or to use plain IFs for it.
Since I don't have yet a solution for this and it is not a so urgent task I will rollback this commit and I will add a task for plural support.